comicScraper.py

```diff
@@ -180,10 +210,23 @@ def main():
 
     # set variables from arguments
     startURL = arguments.URL
+    singleIssue = False
+    if "?id=" in startURL:
+        singleIssue = True
+        print("You have provided the link for a single issue.")
+        print("When providing the URL for the info page, this script will download all issues in the series.")
+
     if arguments.folder != None:
+        print(f"Title detected, using {arguments.folder} as title")
         comicTitle = arguments.folder
     else:
-        comicTitle = getComicTitle(startURL)
+        print("No title specified. Reading title from Url...")
+        comicTitle = getComicTitle(startURL, singleIssue)
+        print("Using title: " + comicTitle)
```

Comment:
probably want to swap to fstrings here (.format would also work, less up to date though). 
```
print(f"Using title: {comicTitle}")
```

Reply:
copilot wrote this for me lol

Reply:
lmao, yeah this isn't standard practice anymore. in python 2 you had to build strings with `+`, then later they added "string {}".format(value_for_the_bracket), then in a later version of 3 they added fstrings `f" things {value}"`

whenever possible use fstrings

Reply:
Updated